x-pack/plugins/spaces/public/management/edit_space/edit_space_roles_tab.tsx

```diff
@@ -99,7 +100,7 @@ export const EditSpaceAssignedRolesTab: FC<Props> = ({ space, features, isReadOn
               }}
             />
           </EditSpaceProvider>,
-          { theme, i18n: i18nStart }
+          { theme, i18n: i18nStart, userProfile }
```

Comment:
Not using a single large object with the services here, because this `toMountPoint` is within a `useCallback`. See https://github.com/elastic/kibana/pull/202606#discussion_r1871588439